jsonpaths/com.snowplowanalytics.snowplow/mobile_context_1.json

```diff
@@ -17,6 +17,8 @@
     "$.data.deviceManufacturer",
     "$.data.deviceModel",
     "$.data.carrier",
+    "$.data.networkType",
+    "$.data.networkTechnology",
```

Comment:
These fields need to be appended to the bottom of the file as they are additive changes.